build-instruction.md

```diff
@@ -452,7 +452,8 @@ The software was validated on:
 
 ### Software Requirements
```

Comment:
Please update info about Python requirements(Python3.5 -> Python3.6) because of end of support Python3.5

Reply:
thanks, fixed